examples/armv4t/gdb/exec_file.rs

```diff
@@ -0,0 +1,23 @@
+use gdbstub::common::Pid;
+use gdbstub::target;
+use gdbstub::target::TargetResult;
+
+use crate::emu::Emu;
+
+impl target::ext::exec_file::ExecFile for Emu {
+    fn get_exec_file(
+        &self,
+        _pid: Option<Pid>,
+        offset: usize,
+        length: usize,
+        buf: &mut [u8],
+    ) -> TargetResult<usize, Self> {
+        let filename = b"/test.elf";
+        let len = filename.len();
+        let data =
+            &filename[(offset as usize).min(len) as usize..((offset + length) as usize).min(len)];
```

Comment:
this can now be simplified to

```suggestion
            &filename[offset.min(len)..(offset + length).min(len)];
```